src/services/DataHandler.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { createContext, useState, useEffect, useContext } from 'react';
+
+// Create a context to provide the data
+const DataContext = createContext();
+
+export function useData() {
+    return useContext(DataContext);
+}
+
+async function fetchKeywordInfoData() {
+  const url = 'http://192.168.0.112:5080/keyword-abilities';
```

Comment:
We don't want to hard code addresses. Could we modify this so the address and port are fetched in any of the following ways instead?
- a config file
- as an argument when launching the site
- from an environment variable?

Reply:
Made an enviroment file for the API addresses. We should probably add everything for flask, the DB and server as well. I've added an env.example as a preset. I'm gonna look into adding stuff to our README later with some instructions and examples for the .env values. Could be useful if we move all of /database/ and /server/ into .env. :smiley: